thewalrus/_permanent.py

```diff
@@ -15,29 +15,21 @@
 Permanent Python interface
 """
```

Comment:
This need to have a little bit more documentation as for example in lines 14-36 of https://github.com/XanaduAI/thewalrus/blob/master/thewalrus/fock_gradients.py